packages/notifications/src/Alert.js

```diff
@@ -31,16 +32,17 @@ const Alert = styled(Well).attrs({
       // Validation types
       [CalloutStyles['c-callout--success']]: props.type === VALIDATION.SUCCESS,
       [CalloutStyles['c-callout--warning']]: props.type === VALIDATION.WARNING,
-      [CalloutStyles['c-callout--error']]: props.type === VALIDATION.ERROR
+      [CalloutStyles['c-callout--error']]: props.type === VALIDATION.ERROR,
+      [CalloutStyles['c-callout--info']]: props.type === VALIDATION.INFO,
+      [CalloutStyles['c-callout--recessed']]: props.type === VALIDATION.INFO
     })
 })`
   ${props => retrieveTheme(COMPONENT_ID, props)};
 `;
 
 Alert.propTypes = {
-  recessed: PropTypes.bool,
-  floating: PropTypes.bool,
-  type: PropTypes.oneOf([VALIDATION.SUCCESS, VALIDATION.WARNING, VALIDATION.ERROR]).isRequired
+  type: PropTypes.oneOf([VALIDATION.SUCCESS, VALIDATION.WARNING, VALIDATION.ERROR, VALIDATION.INFO])
```

Comment:
Why did we remove the existing propTypes? It still gets passed through to the Well component.

Reply:
They are not valid for the `<Alert>` component – there should never be a `floating` alert (and `recessed` isn't applicable). Note given the styled components extension, this only affects documentation. Same for `recessed` on `<Notification>` below.

Reply:
p.s. @allisonacs approved ™️

Reply:
👍